config/100-namespace.yaml

```diff
@@ -18,4 +18,7 @@ metadata:
   name: kourier-system
   labels:
     networking.knative.dev/ingress-provider: kourier
+    app.kubernetes.io/name: kourier
```

Comment:
unsure if you want to be consistent with the other repos and call this `net-kourier`

Reply:
+1 for `net-kourier`.